imx/examples/workflows/main.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"context"
+	"log"
+	"strings"
+
+	"github.com/immutable/imx-core-sdk-golang/imx"
+	"github.com/immutable/imx-core-sdk-golang/imx/api"
+	"github.com/immutable/imx-core-sdk-golang/imx/signers/ethereum"
+	"github.com/immutable/imx-core-sdk-golang/imx/signers/stark"
+	"github.com/joho/godotenv"
+)
+
+func main() {
+	var envs map[string]string
+	envs, err := godotenv.Read("../.env")
+	if err != nil {
+		log.Fatalf("Error loading .env file: %v", err)
+	}
+
+	alchemyAPIKey := envs["ALCHEMY_API_KEY"]
+	signerPrivateKey := envs["OWNER_ACCOUNT_PRIVATE_KEY"]
+	enableDebugLogging := strings.EqualFold(envs["DEBUG_LOGGING"], "true")
+
+	configuration := api.NewConfiguration()
+	// Enable debug logging.
+	if enableDebugLogging {
+		configuration.Debug = true
+	}
+
+	// Using context value to switch/specify the server before sending request.
+	// If nothing is specified, the default server will be used which will be first one in the open api spec list.
+	ctx := context.TODO()
+
+	cfg := imx.Config{
+		APIConfig:   configuration,
+		EthereumRPC: "https://eth-ropsten.alchemyapi.io/v2/" + alchemyAPIKey,
```

Comment:
Should this url be part of config Environment constants?

Reply:
We will need to pass in the alchemyApiKey value, that is how it was previously until the changes in this PR.

Reply:
Addressed